### DISCUSSION TITLE
How to open a single file using the ZED's UI ?

### DISCUSSION THREAD

**Discussion by llagerlof** [zed-industries/zed#14509] [Feature Requests]
Normally it's the `File > Open` menu item in normal text editors (or `:e .` in vim).

**Comment by tepavcevic** [comment]
You can do it from the terminal via `zed example_file`
If you are on a linux distro that has a named the binary differently you'll need to adjust the command accordingly (e.g. `zed-editor` or similar)

**Reply by SomeoneToIgnore** [reply]
Also, at least on macOS drag and drop works.

**Reply by llagerlof** [reply]
Thanks, but I need to know if the user can open a file using an embedded feature in the UI.

**Reply by KaKi87** [reply]
At least you can use Zed's terminal rather than an external one, but agreed. ^^

**Comment by caiigee** [comment]
Why not create a [Feature Request](https://github.com/zed-industries/zed/issues/new?assignees=&labels=admin+read%2Ctriage%2Cenhancement&projects=&template=0_feature_request.yml) for this? It would be awesome if Zed had some kind of dialog to open files with. Currently there is only the ability to open folders with `Ctrl+O` which creates an "Open Folder" window. It would be cool if we could open a file using that window aswell.

**Comment by donpdonp** [comment]
is this question being misunderstood? I came here because I cannot find a way to open an editor with a file from the open project using the keyboard. A seemingly simply task but I can only do it with the mouse. Ctrl-O opens an entire folder, I just want to pick from the files already in the project. I've gone through the keybinding names but nothing looks applicable.

**Reply by yancoupro** [reply]
finally found it:
"cmd-p": "file_finder::Toggle"

**Comment by KobusG** [comment]
This is necessary as in SSH remote connection there is no 'zed' in the path, and there is no way to open a single out-of-workspace file with cmd-p

**Comment by neumaneuma** [comment]
As of macOS version 0.163.3 I have a `File->Open` menu item.

**Reply by KaKi87** [reply]
I have 0.164.2 on Linux, and no `File` item in the menu.

**Reply by llagerlof** [reply]
That's strange. In the same build there are different features for each OS.

**Reply by wlinna** [reply]
I'm using Zed 0.168.2 on Linux, I have `File->Open`, but `File->Open` only allows opening folders. But I don't want to change the folder

**Reply by KaKi87** [reply]
Oh, it appears only after clicking the hamburger. That's very bad design.

**Comment by wlinna** [comment]
Does Zed have a way of opening a single file at all outside the open workspace? Because after trying different options, the only GUI way to do it seems to be to drag and drop the file on Zed

**Reply by SomeoneToIgnore** [reply]
https://github.com/zed-industries/zed/pull/22625 was merged 5 days ago and should come out in Zed 0.170

**Reply by tredondo** [reply]
> #22625 was merged 5 days ago and should come out in Zed 0.170

Looks like there's an issue with the hotkey for opening a new file. Sometimes Ctrl-P works, other times it doesn't and I have to use Ctrl-Shift-N.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Reply by tredondo** [reply]
Discord is not searchable or indexable. Why silo the knowledge there and have the same Q&A pop up again and again?
